agent-inject/agent/agent.go

```diff
@@ -636,6 +645,10 @@ func (a *Agent) Patch() ([]byte, error) {
 				a.ContainerVolumeMounts(),
 				fmt.Sprintf("/spec/initContainers/%d/volumeMounts", i))...)
 		}
+
+		// Add shareProcessNamespace
+		a.Patches = append(a.Patches, updateShareProcessNamespace(
```

Comment:
looks like this might need to be changed

Reply:
Yup, fixed in 468c33bc225ff4377aea4feb9fc6f986ce96f39a